src/foundations/theme-provider/global-styles/globalStyles.ts

```diff
@@ -9,12 +9,15 @@ import VElevation from './variables.elevation';
 import VFontFaces from './variables.font-faces';
 import VFontStyle from './variables.font-style';
 import VBorderRadius from './variables.border-radius';
+import reset from './reset-styles';
 
 type PGlobalStyles = {
     theme: TTheme;
 };
 
 const GlobalStyle = createGlobalStyle`
+    ${reset};
```

Comment:
When would we want to reset the global styles?

Reply:
this is to reset all styles that come from the browsers.